recipes/onetbb/all/conanfile.py

```diff
@@ -53,8 +78,15 @@ def configure(self):
             self.options.rm_safe("fPIC")
         else:
             del self.options.tbbproxy
+            self.options.rm_safe("tbbbind")
         if not self.options.tbbmalloc:
             self.options.rm_safe("tbbproxy")
+        if self._tbbbind_explicit_hwloc:
+            self.options["hwloc"].shared = True
```

Comment:
You still need to validate it in validate(), because users can override that option value.

```
def validate(self):
    if self._tbbbind_explicit_hwloc and not self.dependencies["hwloc"].options.shared:
        raise ConanInvalidConfiguration(f"{self.ref} requires hwloc:shared=True to be built.")
```

Reply:
added to `validate_build`

Reply:
missing change

Reply:
please, see `validate_build`

Reply:
addressed here https://github.com/conan-io/conan-center-index/pull/19652